STATUS_GO_VERSION

```diff
@@ -1,3 +1,3 @@
 ## DO NOT EDIT THIS FILE BY HAND. USE `scripts/update-status-go.sh <tag>` instead
 
-v0.30.0-beta.0
+develop
```

Comment:
You shouldn't be using a branch as a `status-go` version, look above, the usage of the script is described:
```
## DO NOT EDIT THIS FILE BY HAND. USE `scripts/update-status-go.sh <tag>` instead
```
Please use a tag that will not change what it points at. Otherwise the same build will end up with different hashes.

Reply:
that's literally what I did, I used the script and the script updated this file